util/nrt_diff/nrtDiff.py

```diff
@@ -1265,17 +1265,20 @@ def XMLCloseEnough (path_left, path_right):
 def DTWDistance(left, right) :
     n = len(left)
     m = len(right)
-    DTW = [[0 for j in range(m+1)] for i in range(n+1)]
-
-    for i in range(1, n+1):
-        for j in range (1, m+1):
-            DTW[i][j] = 999999
+    if n == m:
+        if left == right:
+            return 0.
+    DTW = [[0 if (j == 0 and i == 0) else 999999 for j in range(m+1)] for i in range(n+1)]
```

Comment:
shouldn't it be j == 0 or i == 0 ? ( first line or first column)

Reply:
yes, sorry again :)